pkg/controller/siddhiprocess/config.go

```diff
@@ -81,10 +81,10 @@ const (
 	MaxUnavailable             int32  = 0
 	MaxSurge                   int32  = 2
 	HealthPort                 int32  = 9090
-	ReadyPrPeriodSeconds       int32  = 20
-	ReadyPrInitialDelaySeconds int32  = 60
-	LivePrPeriodSeconds        int32  = 120
-	LivePrInitialDelaySeconds  int32  = 120
+	ReadyPrPeriodSeconds       int32  = 10
```

Comment:
Do we need int32 here? Can't we just use int?

Reply:
Since K8s structs that uses this parameter need `int32`. If we use int we need to execute an unnecessary conversion step.